### DISCUSSION TITLE
Whenever you fork a repository it doesn't copy topics.It would be nice to have feature.

### DISCUSSION THREAD

**Discussion by harsha549** [community/community#23277] [New to GitHub]
<p>Suppose I have just forked the github repository like Apache Airflow</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/051206a4297ad0ad6131cd8f8c977419b340be0c.png" data-download-href="/uploads/short-url/IR0tOpAw087YxIUy6ubWX85dg0.png?dl=1" title="githubwithTopics.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/15097959/181076010-24beb968-031b-4cd7-b96f-b41e5805c9cf.png" alt="githubwithTopics.PNG" data-base62-sha1="IR0tOpAw087YxIUy6ubWX85dg0" width="690" height="165" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/051206a4297ad0ad6131cd8f8c977419b340be0c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">githubwithTopics.PNG</span><span class="informations">903×216 15.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>My repo is shown below </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/862fe21dbabfcd21283e408ff8f31736a4b49b6c.png" data-download-href="/uploads/short-url/j94zmRIyAXPppQXQcJjv3NClkpS.png?dl=1" title="githubWithoutTopics.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/15097959/181076011-52a8fa04-7953-4abc-9f07-234d37459a20.png" alt="githubWithoutTopics.PNG" data-base62-sha1="j94zmRIyAXPppQXQcJjv3NClkpS" width="690" height="125" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/862fe21dbabfcd21283e408ff8f31736a4b49b6c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">githubWithoutTopics.PNG</span><span class="informations">986×180 13.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If you observe the topics aren’t copied during fork.It would awesome if the topics also get copied.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams. Thanks again for reaching out :grinning:</p>
